modules/ossm-about-configuring-a-gateway-to-accept-ingress-traffic.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+
+// gateways/ossm-about-gateways.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-configuring-a-gateway-to-accept-ingress-traffic_{context}"]
+= About configuring a gateway installed using gateway injection to accept ingress traffic
+:context: ossm-about-configuring-a-gateway-to-accept-ingress-traffic
+
+When you install a gateway using gateway injection you can configure it to receive ingress traffic using the {istio} `Gateway` and `VirtualService` resources in combination. The {istio} `Gateway` resource describes a load balancer operating at the edge of the mesh that receives incoming or outgoing HTTP/TCP connections. The `Gateway` specification describes a set of ports that should be exposed, the type of protocol to use, and the Server Name Indication (SNI) configuration for the load balancer. `VirtualServices` define routing rules to apply to an {istio} `Gateway`, similar to how you can use `VirtualServices` to define routing rules for internal mesh traffic.
+
```

Comment:
Deploy a bookinfo application into a mesh namespace.

oc create ns bookinfo 
oc label namespace bookinfo istio-injection=enabled
 oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo

Reply:
We discussed and don't think we need this information in this topic.